junifer/data/utils.py

```diff
@@ -5,14 +5,24 @@
 # License: AGPL
 
 from collections.abc import MutableMapping
+from pathlib import Path
 from typing import Optional, Union
 
 import numpy as np
 
-from ..utils import logger, raise_error
+from ..utils import config, logger, raise_error
 
 
-__all__ = ["closest_resolution", "get_native_warper"]
+__all__ = [
+    "JUNIFER_DATA_VERSION",
+    "closest_resolution",
+    "get_dataset_path",
+    "get_native_warper",
+]
+
+
+# junifer-data version constant
+JUNIFER_DATA_VERSION = "1.0.0"
```

Comment:
I would keep this not semver. Just incremental tags. Also easier to handle in case we want to "clean" the directory. Any new modification to the junifer-data package is a new version.

Reply:
No problem with no semver and just int. Not sure about the rationale of easier "clean".